docker/generic/start_proxy.py

```diff
@@ -1525,6 +1544,11 @@ def start_envoy(args):
     cm_proc = start_config_manager(gen_proxy_config(args))
     envoy_proc = start_envoy(args)
 
+    pid_list.append(cm_proc.pid)
+    pid_list.append(envoy_proc.pid)
+    signal.signal(signal.SIGTERM, sigterm_handler)
+    signal.signal(signal.SIGINT, sigterm_handler)
```

Comment:
Why not handle all signals that envoy handles?

https://github.com/envoyproxy/envoy/blob/3752119b94ae4a5ff337364f585f54305406dc7f/source/server/server.cc#L849-L873

Reply:
Nevermind, we don't have hot restart